backend/question-service/routes/question.routes.js

```diff
@@ -4,7 +4,7 @@ module.exports = app => {
   var router = require("express").Router();
 
   // Create a new question
-  router.post("/auth/", questions.create);
+  router.post("/", questions.create);
```

Comment:
Why is `auth` removed here? Does it still route to use `auth_request` in the gateway?